src/Microsoft.Extensions.DependencyInjection.Abstractions/ServiceLifetime.cs

```diff
@@ -3,10 +3,22 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Specifies the lifetime of a service in an <see cref="IServiceCollection"/>.
+    /// </summary>
     public enum ServiceLifetime
     {
+        /// <summary>
+        /// Specifies that a single instance of the service will be created.
+        /// </summary>
         Singleton,
+        /// <summary>
+        /// Specifies that a new instance of the service will be created for each scope.
```

Comment:
I think it could be helpful to point out that in ASP.NET scopes are normally created per request.